### PR TITLE
[bug 1194351] Make static HTML localizable

### PR DESCRIPTION
**Pull request by rehandalal**
phew!!

r?

### CODE REVIEW

**Comment by mythmon** [comment]
For some reason I can't comment on individual lines.

About `i18n.js`: how does that relate to the already existing [`l10n.js`](https://github.com/mozilla/buddyup/blob/9acf86651442106df4e1ba9a90cdbe6aac7b3795/app/js/l10n.js)? l10n.js defines some gettext and ngettext functions that include formatting directly, and are used already in some of the templates.

File: app/views/main/question_list_helper.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      </li>
+    </ul>
+
+    <div id="panel_new" class="bb-tabpanel questionList" role="tabpanel" aria-labelledby="tab1">
+      <section id="myquestions" class="questions js-question-container all" data-all="true"></section>
+      <a href="#" role="button" id="load-more-my-questions" data-icon="download" class="ql-loadMore hide">{{ _('Load more') }}</a>
+      <div id="refresh-toolbar" class="ql-bottomToolbar hide">
+        <a href="#" role="button" id="refresh-my-questions" class="ql-refresh" data-icon="sync"></a>
+              <span>{{ _('Last sync:') }}
+                <time id="last-refreshed-time">{{ _('x hours ago') }}</time>
```
Comment (mythmon) [inline app/views/main/question_list_helper.html]: I think this is going to be kind of goofy and not-localization friendly. Also, not RTL friendly.

**Comment by mythmon** [comment]
I think this is probably going to have some weird localization things, but in general it is a huge first step. I think we need to get rid of one of l10n.js or i18n.js. We can probably pass on that RTL issue I pointed out. Besides that, I think this looks fine. I'm going to pull it down and play with it in the simulator.

**Comment by mythmon** [comment]
My simulator is broken, so I tried this in a normal browser. The profile page isn't loading for me, I just a white screen. Nothing in the console. It works on master. Does it work for you?

**Comment by mythmon** [comment]
This fixups fixed the profile page for me.

Also, I just realized why my token stopped working, and why I had to clear everything: Sumo stage got updated today, so the tokens I had were all wrong. Maybe that was one of your problems too?

I think this is still a problem with duplicated gettext and friends in l10n.js and i18n.js.

**Comment by rehandalal** [comment]
updated ^

**Comment by mythmon** [comment]
That works. Woot.

**Pull request merged**